zebedee-cms/src/main/java/com/github/onsdigital/zebedee/configuration/CMSFeatureFlags.java

```diff
@@ -39,6 +40,7 @@ public class CMSFeatureFlags {
     private final boolean isJwtSessionsEnabled;
 
     private final boolean isKafkaEnabled;
+    private final boolean isStaticFilesPublishingEnabled;
```

Comment:
I don't know why an empty line is added in each `private final` but I would suggest to either remove those empty lines or to add that extra empty line for consistently